e2e/cypress/integration/enterprise/guest_accounts/system_console_manage_guest_spec.js

```diff
@@ -129,7 +129,7 @@ describe('Guest Account - Verify Manage Guest Users', () => {
         verifyGuest();
     });
 
-    it('MM-18048 Change Email of a Guest User and Verify', () => {
+    it('MM-T1394 Change Email of a Guest User and Verify', () => {
```

Comment:
Please check again. MM-T1394 should consider allowed list for guest domains which seemed to be not in the script.

Reply:
Reverted for now.